src/components/Form/Label/Label.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export function Label({ labelName, labelFor }) {
+  return (
+    <label className='form-label' htmlFor={labelFor}>
```

Comment:
appreciation - you are using the htmlFor attribute, it is a good practice

Reply:
> Nice work 💯 The code is really good, you have broken down everything into small components.
> 
> suggestion - you can add the password show/hide functionality using useState & increase the size of show/hide icon so that it's easy to click.

Thanks I will increase the font-size of icons. Regarding show-hide functionality, right now I am converting my HTML screens to React so that's why I have not added any state variable or functionality.